python/ray/data/block.py

```diff
@@ -163,6 +164,16 @@ def __init__(self):
         # differentiate from previous tasks on the same worker.
         self.max_rss_bytes: int = 0
 
+    def add(self, other: "BlockExecStats"):
+        """Add the other BlockExecStats into this BlockExecStats, by combining
+        each statistic.
+        """
+        if other.wall_time_s:
+            self.wall_time_s = float(self.wall_time_s or 0) + other.wall_time_s
```

Comment:
This has an underlying requirement that those BlockExecStats are executed serially. Can you make that clear at the comment or the API naming? Basically this cannot aggregate block stats across tasks.

Reply:
@jianoaix - sure, added comment.